deployments/kubehound/docker-compose.ingestor.yaml

```diff
@@ -0,0 +1,14 @@
+version: "3.8"
```

Comment:
version or not version, this is the question? (you removed it from the other ones, soo....)

Reply:
seems deprecated:
* The top-level version property is defined by the Compose Specification for backward compatibility. It is only informative you'll receive a warning message that it is obsolete if used.

ref: https://github.com/compose-spec/compose-spec/blob/master/spec.md#version-top-level-element-obsolete

removing it